packages/openapi-generator/src/generator.ts

```diff
@@ -76,6 +79,8 @@ async function generateSDKSources(
   await createFile(serviceDir, 'api.ts', apiFile(openApiDocument), true);
   await createFile(serviceDir, 'index.ts', indexFile(), true);
   if (options.generatePackageJson) {
+    logger.info(`Generating package.json in ${serviceDir}.`);
```

Comment:
Minor: Isn't that rather debug level

Reply:
I'm fine with `debug`, but currently most of the cases we use `info`, perhaps we should reconsider all the log level again for alignment.